### PR TITLE
Optimize script when create stream cert

### PR DESCRIPTION
**Pull request by gujun4990**
Signed-off-by: jun.gu <gujun4990@outlook.com>

### CODE REVIEW

**Comment by gy95** [comment]
welcome @gujun4990 
need sign off the commit to pass the DCO 😄 
ref: https://github.com/kubeedge/kubeedge/pull/3412/checks?check_run_id=4364943896

File: build/tools/certgen.sh [hunk 1]
```diff
@@ -67,8 +67,9 @@ stream() {
     readonly STREAM_KEY_FILE=${certPath}/stream.key
     readonly STREAM_CSR_FILE=${certPath}/stream.csr
     readonly STREAM_CRT_FILE=${certPath}/stream.crt
-    readonly K8SCA_FILE=/etc/kubernetes/pki/ca.crt
-    readonly K8SCA_KEY_FILE=/etc/kubernetes/pki/ca.key
+
+    readonly k8sCaFile=${K8SCA_FILE:-/etc/kubernetes/pki/ca.crt}
+    readonly k8sCaKeyFile=${K8SCA_KEY_FILE:-/etc/kubernetes/pki/ca.key}
```
Comment (gy95) [inline build/tools/certgen.sh:72]: why need to fix this? IMHO, K8SCA_FILE is never set. :)

Comment (gujun4990) [reply build/tools/certgen.sh:72]: Consider the path of k8sCA maybe different, so optimize the code. If the file is never set, maybe the patch is not necessary.

Comment (zc2638) [reply build/tools/certgen.sh:72]: Maintain uniform variable naming rules?

Comment (gujun4990) [reply build/tools/certgen.sh:72]: Modify k8sCaFile to K8SCA_FILE, k8sCaKeyFile to K8SCA_KEY_FILE? Like this?

Comment (zc2638) [reply build/tools/certgen.sh:72]: yeah, You can also change it all into humps.

Comment (gujun4990) [reply build/tools/certgen.sh:72]: OK, I will fix this.

File: build/tools/certgen.sh [hunk 2]
```diff
@@ -67,8 +67,9 @@ stream() {
     readonly STREAM_KEY_FILE=${certPath}/stream.key
     readonly STREAM_CSR_FILE=${certPath}/stream.csr
     readonly STREAM_CRT_FILE=${certPath}/stream.crt
-    readonly K8SCA_FILE=/etc/kubernetes/pki/ca.crt
-    readonly K8SCA_KEY_FILE=/etc/kubernetes/pki/ca.key
+
+    readonly K8SCA_FILE=${K8SCA_FILE:-/etc/kubernetes/pki/ca.crt}
+    readonly K8SCA_KEY_FILE=${K8SCA_KEY_FILE:-/etc/kubernetes/pki/ca.key}
```
Comment (gy95) [inline build/tools/certgen.sh:72]: this is redundant, just readonly K8SCA_FILE=/etc/kubernetes/pki/ca.crt, K8SCA_FILE is first defined.

File: build/tools/certgen.sh [hunk 3]
```diff
@@ -84,7 +85,7 @@ stream() {
         SUBJECTALTNAME="${SUBJECTALTNAME}IP.${index}:${ip}"
     done
 
-    cp /etc/kubernetes/pki/ca.crt ${caPath}/streamCA.crt
+    cp ${K8SCA_FILE} ${caPath}/streamCA.crt
```
Comment (gy95) [inline build/tools/certgen.sh:88]: this is reasonable :)

File: build/tools/certgen.sh (hunk 2 above)
Comment (gujun4990) [reply build/tools/certgen.sh:72]: If having the code, It's necessary to set K8SCA_FILE and K8SCA_KEY_FILE environment.

Comment (gy95) [reply build/tools/certgen.sh:72]: WDYT @zc2638

**Review by fisherxu — COMMENTED** [review]
/approve
Leave @zc2638  for final check.

File: build/tools/certgen.sh (hunk 2 above)
Comment (zc2638) [reply build/tools/certgen.sh:72]: It makes sense, and I am using it too. 
Many private environments, their certificates are not in the default k8s path.
The script is executed by changing the required path by defining variables.
@gy95

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**